ca_gazebo/src/model_pose_publisher_plugin.cpp

```diff
@@ -0,0 +1,80 @@
+#include <ros/ros.h>
```

Comment:
Is this the plugin that you're using to detect whether the robot is docker or not?